### PR TITLE
Rename library and logger label

### PR DESCRIPTION
**Pull request by bkaney**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [![Build Status](https://travis-ci.org/Vermonster/node-fhir-client.svg?branch=master)](https://travis-ci.org/Vermonster/node-fhir-client) [![Coverage Status](https://coveralls.io/repos/github/Vermonster/node-fhir-client/badge.svg?branch=master)](https://coveralls.io/github/Vermonster/node-fhir-client?branch=master)
```
Comment (anaerobeth) [inline README.md]: Does the reference to `node-fhir-client` in the badges need updating as well?

Comment (bkaney) [reply README.md]: yes! I'll update it

File: README.md [hunk 2]
```diff
@@ -83,16 +83,19 @@ See the [SMART Launch](./examples/smart-launch.js) example with OAuth2 support.
...
+$ DEBUG=fhir-kit-client:* node smart-launch.js
 ```
 
 # Todo
 
+
+
 # Contributing
 
 All are welcome to contribute. By participating in this project you agree to
```
Comment (anaerobeth) [inline README.md]: The url for `Code of Conduct` link in the next line of the Readme needs updating as well.

**Comment by anaerobeth** [comment]
Might also need to regenerate docs to replace references to the old repo name:

```
docs/fhir-client/0.1.0/index.html
51:        <article><p><a href="https://travis-ci.org/Vermonster/node-fhir-client"><img src="https://travis-ci.org/Vermonster/node-fhir-client.svg?branch=master" alt="Build Status"></a> <a href="https://coveralls.io/github/Vermonster/node-fhir-client?branch=master"><img src="https://coveralls.io/repos/github/Vermonster/node-fhir-client/badge.svg?branch=master" alt="Coverage Status"></a></p>
```

**Comment by bkaney** [comment]
> Might also need to regenerate docs to replace references to the old repo name:

Yea, I will do that at the end in case there are other updates I missed.

**Pull request merged**